packages/webapp/src/store/api/apiSlice.ts

```diff
@@ -49,7 +62,21 @@ export const api = createApi({
       query: () => `${animalGroupsUrl}`,
       providesTags: ['AnimalGroups'],
     }),
+    getDefaultAnimalTypes: build.query<DefaultAnimalType[], string>({
```

Comment:
One small thing I noticed when combining this with @Duncan-Brain's inventory branch is that `getDefaultAnimalTypes` will also be called without the `count` param at all, so the `QueryArg` type of this one should be `string | void` to cover both cases -- not sure which branch that is easier reflected on.

Reply:
Thank you, I will update my PR.
I'm just wondering, does it give you an error??

Reply:
Thank you! 🙏  Yes the type checker error was in Duncan's `useAnimalInventory`:

![Screenshot 2024-03-05 at 9 44 03 AM](https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/a5962ba2-bb68-4c2d-98dc-986628580f94)

Reply:
Ah! I am using `useQueries` so it didn't give me an error!
Thank you Joyce!!